packages/tinacms/src/toolkit/react-sidebar/components/VersionInfo.tsx

```diff
@@ -2,11 +2,9 @@ import { useCMS } from '@toolkit/react-tinacms';
 import formatDistanceToNow from 'date-fns/formatDistanceToNow';
 import { Check, LoaderCircle, TriangleAlert } from 'lucide-react';
 import React from 'react';
-// import { version as currentVersion } from '../../../../package.json';
+import { version as currentVersion } from '../../../../package.json';
```

Comment:
will this find the users package.json correctly?

Reply:
It was how we were displaying the version number before, its a reference to the tinacms package.json